### ISSUE TITLE
sasl PHP Fatal error

### ISSUE THREAD

**Issue by Dark345** [litespeedtech/lsmcd#28]
Funny you should mention that.  We've been doing some preliminary testing that seems to address the WP Cache problem by installing the libmemcached-awesome library and completely replacing the existing libmemcached library with it.  It seems to have fixed the problem.  If you don't mind giving something like that a try, we have an early cheat-sheet of instructions for it.  We'd love to hear your experiences with it.  Check out: https://docs.google.com/document/d/1GNZ3EOcOeBqYS19rl5rnGoh8I9N1BFU3htSn_IPaavg/edit?usp=sharing  
Thanks,

Bob

_Originally posted by @rperper in https://github.com/litespeedtech/lsmcd/issues/26#issuecomment-1219537336_
      

Hello! is this solved in v1.4.33 ?

https://docs.litespeedtech.com/lscache/lscwp/cache/#method

**Comment by rperper** [comment]
No I'm afraid not.  The problem has to be solved in PHP.  There's still no generally available fix at this point, but we're still working through ideas.
Thanks,

Bob

**Comment by m6w6** [comment]
Let me know if I could help in any way wrt libmemcached-awesomized or PHP

**Comment by rperper** [comment]
Nice to meet you - love your product!  libmemcached was truly broken until you fixed it.  The problem for us is getting all of the pieces together: libmemcached-awesome of course, but also the PHP pear package (3.2.0) a PHP distribution built to use it and finally our web server library (lsphp) all on a distribution we can work on.  It was a bear finally isolating the bug in PHP memcached's support to libmemcached and it's even worse finding a way to apply your lovely fixes.  If you have any thoughts about a full environment with all of this stuff in it that a customer can use, we'd sure appreciate it.  Thanks!

Bob

**Comment by Dark345** [comment]
hello @rperper is this fixed in v1.4.34 ??

**Comment by rperper** [comment]
No, not yet.  We're hoping to get libmemcached-awesome to be included in lsphp, but it's not in there yet.  Sorry, but we haven't forgotten.

**Comment by Dark345** [comment]
> No, not yet. We're hoping to get libmemcached-awesome to be included in lsphp, but it's not in there yet. Sorry, but we haven't forgotten.

Is the libmemcached-awesome the solution?

[here ](https://github.com/litespeedtech/lsmcd/issues/26#issuecomment-1219856345)I remember trying libmemcached-awesome without success, you even followed my case privately

**Comment by rperper** [comment]
We're pretty sure it is.  The only way to be 100% sure is to follow the whole thing with debugging statements added (can't really be added to a debugger because it's way out of what debuggers can follow).  libmemcached-awesome fixes the problem in the PHP extension that supports memcached (which uses the conventional libmemcached library).  Once we get a lsphp that has libmemcached-awesome support embedded, then we can try it for sure.
